[2025-01-06T23:42:39.645+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_insert_log.ft_posting_f manual__2025-01-06T23:42:13.208216+00:00 [queued]>
[2025-01-06T23:42:39.791+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_insert_log.ft_posting_f manual__2025-01-06T23:42:13.208216+00:00 [queued]>
[2025-01-06T23:42:39.792+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-01-06T23:42:40.068+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): ft_posting_f> on 2025-01-06 23:42:13.208216+00:00
[2025-01-06T23:42:40.107+0000] {standard_task_runner.py:60} INFO - Started process 673 to run task
[2025-01-06T23:42:40.215+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_insert_log', 'ft_posting_f', 'manual__2025-01-06T23:42:13.208216+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/dag_insert_log.py', '--cfg-path', '/tmp/tmp3qzt2mmd']
[2025-01-06T23:42:43.368+0000] {standard_task_runner.py:88} INFO - Job 39: Subtask ft_posting_f
[2025-01-06T23:42:44.226+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_insert_log.ft_posting_f manual__2025-01-06T23:42:13.208216+00:00 [running]> on host 22295d32c47d
[2025-01-06T23:42:45.489+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='saida' AIRFLOW_CTX_DAG_ID='dag_insert_log' AIRFLOW_CTX_TASK_ID='ft_posting_f' AIRFLOW_CTX_EXECUTION_DATE='2025-01-06T23:42:13.208216+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-06T23:42:13.208216+00:00'
[2025-01-06T23:42:45.710+0000] {base.py:83} INFO - Using connection ID '***-db' for task execution.
[2025-01-06T23:42:51.212+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-06T23:42:51.248+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_insert_log, task_id=ft_posting_f, execution_date=20250106T234213, start_date=20250106T234239, end_date=20250106T234251
[2025-01-06T23:42:51.377+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-06T23:42:51.432+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
